contributors/devel/sig-scheduling/scheduler_framework_plugins.md

```diff
@@ -0,0 +1,142 @@
+# Scheduler Framework Plugins
+
+## Creating a new in-tree plugin
+
+Read [the docs](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/)
+to understand the different extension points within the scheduling framework.
+
+TODO([#95156](https://github.com/kubernetes/kubernetes/issues/95156)): flesh this out a bit more.
```

Comment:
migrate this issue to community repo

Reply:
I don't have the perms to migrate issues I think.

Reply:
I can take care of it 👍

Reply:
Done: https://github.com/kubernetes/community/issues/5466

Thanks, @mrbobbytables!